apy/anki.py

```diff
@@ -410,6 +412,12 @@ def add_notes_from_list(self, parsed_notes, tags=''):
 
         return notes
 
+    def add_single_note(self, fields, tags='', model=None, deck=None):
+        if model is not None:
```

Comment:
Minor: All public methods should have a docstring. I could possible amend this when merging.